ros_ws/src/simulation/racer_control/include/drive_param_converter.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include <string>
+#include <cmath>
```

Comment:
<cmath> isn't used in the header file. Move this include to the source file could increase compile time.